docker/app/entrypoint.dev.sh

```diff
@@ -28,5 +28,19 @@ bundle config --global set build.sassc --disable-march-tune-native
 bundle install --quiet || echo "bundle install failed"
 yarn install --silent --frozen-lockfile || echo "yarn install failed"
 
+echo 'Generating .pgpass file from environment variables...'
+cat > ~/.pgpass << EOF
+${DATABASE_HOST:-db}:${DATABASE_PORT:-5432}:*:${DATABASE_USER:-postgres}:${DATABASE_PASS:-postgres}
+${WAREHOUSE_DATABASE_HOST:-db}:${WAREHOUSE_DATABASE_PORT:-5432}:*:${WAREHOUSE_DATABASE_USER:-postgres}:${WAREHOUSE_DATABASE_PASS:-postgres}
+${HEALTH_DATABASE_HOST:-db}:${HEALTH_DATABASE_PORT:-5432}:*:${HEALTH_DATABASE_USER:-postgres}:${HEALTH_DATABASE_PASS:-postgres}
+${REPORTING_DATABASE_HOST:-db}:${REPORTING_DATABASE_PORT:-5432}:*:${REPORTING_DATABASE_USER:-postgres}:${REPORTING_DATABASE_PASS:-postgres}
+EOF
```

Comment:
Can't this just call `echo bin/generate_pgpass.rb` > .pgpass` (also why are you sticking this in the user's home directory, is that the right place?